### PR TITLE
Background gradients

### PR DESCRIPTION
**Pull request by kazumaki**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            </header>
+            <section class="main-text">
+                <h1>What will your verse be?</h1>
+                Everyone has something to share. See how people are using iPad in their unique ways to add to the world's story.
+            </section>
+            <footer>
+                Watch the "Your Verse" film and explore the stories
+            </footer>
+        </section>
+        <section class="bottom-box one">
```
Comment (Ispirett) [inline index.html]: These images  should be wrapped in `a` tags they are links

**Review by Ispirett — CHANGES_REQUESTED** [review]
Excellent job with this project @kazumaki  :+1: 
image

**See message bellow afer fix submit for re-review**

_**For feature projects please write more descriptive commit messages.**_

File: index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            </header>
+            <section class="main-text">
+                <h1>What will your verse be?</h1>
+                Everyone has something to share. See how people are using iPad in their unique ways to add to the world's story.
+            </section>
+            <footer>
+                Watch the "Your Verse" film and explore the stories
+            </footer>
+        </section>
+        <a class="bottom-box one" href="#">
```
Comment (samgaco) [inline index.html:42]: :+1:

**Review by samgaco — APPROVED** [review]
Hi @kazumaki , the project got the pass :white_check_mark: you may merge to the master branch and submit the project completion form. Congratulations on finishing your first solo project :100: 

------------------------------
Moving on :arrow_down: 
* **[For your next code review requests]**  Add a short summary of changes in Pull Request description

----------------
* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**